databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,107 @@
           }
         }
       }
+    },
+    // map of annotations by 'data type' key -> array of annotations of that type
+    annotations: {
+      type: 'hashtable',
+      hashtableKeysValidator: {
+        mustNotBeEmpty: false,
+        type: 'enum',
+        predefinedValues: [ 'metadata', 'record' ]
+      },
+      hashtableValuesValidator: {
+        type: 'array',
+        required: true,
+        mustNotBeEmpty: true,
+        arrayElementsValidator: {
+          type: 'object',
+          required: true,
+          propertyValidators: {
+            // indicates what type of annotation this is
+            // Only embedded is supported to start
+            type: {
+              type: 'enum',
+                immutable: true,
+                required: true,
+                predefinedValues: [ 'embedded' ]
+            },
+            // a simple tag describing what type data this annotation is
+            // example:  'user-metadata', 'ocr-analysis', 'model-XYZ-classification'
+            dataType: {
+              type: 'enum',
+                immutable: true,
+                required: true,
+                predefinedValues: [ 'metadata', 'record' ]
+            },
+            // payload, pretty much unrestricted at this point
+            data: {
+              type: 'object',
+              required: true
+            },
+            // Array of timestamp and user IDs of modifications to this annotation
+            modifications: {
+              type: 'array',
+              mustNotBeEmpty: true,
+              required: true,
+              arrayElementsValidator: {
+                type: 'object',
+                // immutable: true,  hesitant to enable this.  It would require that clients ensure this array remains sorted by chronologically by timestamp.
+                propertyValidators: {
+                  source: { // source of the annotation
+                    type: 'object',
+                      required: true,
+                      propertyValidators: {
+                      type: {
+                        type: 'string',
```

Comment:
Since this schema is intentionally flexible, it may be wise to restrict it to known values for now as well (ie, `books.user`)